operator/v1/types_ingress.go

```diff
@@ -63,17 +63,7 @@ type IngressControllerSpec struct {
 
 	// endpointPublishingStrategy is used to publish the ingress controller
 	// endpoints to other networks, enable load balancer integrations, etc.
-	//
-	// If empty, the default is based on
-	// infrastructure.config.openshift.io/cluster .status.platform:
-	//
-	//   AWS: LoadBalancerService
-	//   All other platform types: Private
-	//
-	// endpointPublishingStrategy cannot be updated.
-	//
-	// +optional
-	EndpointPublishingStrategy EndpointPublishingStrategy `json:"endpointPublishingStrategy,omitempty"`
+	EndpointPublishingStrategy EndpointPublishingStrategy `json:"endpointPublishingStrategy"`
```

Comment:
Is there a reason not to make this a pointer instead, and then make type required?

Reply:
https://github.com/openshift/api/pull/238